releng/org.eclipse.cdt.lsp.target/org.eclipse.cdt.lsp.target.target

```diff
@@ -11,42 +11,53 @@
-			<unit id="org.eclipse.sdk.feature.group" version="0.0.0" />
+			<unit id="org.eclipse.equinox.executable.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.equinox.sdk.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.sdk.feature.group" version="0.0.0"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/lsp4e/snapshots/" />
-			<unit id="org.eclipse.lsp4e" version="0.0.0" />
-			<unit id="org.eclipse.lsp4e.debug" version="0.0.0" />
+			<repository location="https://download.eclipse.org/lsp4e/snapshots/"/>
+			<unit id="org.eclipse.lsp4e" version="0.0.0"/>
+			<unit id="org.eclipse.lsp4e.debug" version="0.0.0"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/mylyn/docs/releases/3.0.42/" />
-			<unit id="org.eclipse.mylyn.wikitext_feature.feature.group" version="0.0.0" />
+			<repository location="https://download.eclipse.org/mylyn/docs/releases/3.0.42/"/>
+			<unit id="org.eclipse.mylyn.wikitext_feature.feature.group" version="0.0.0"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
 			<repository location="https://download.eclipse.org/tm4e/snapshots/"/>
 			<unit id="org.eclipse.tm4e.feature.feature.group" version="0.0.0"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/tools/cdt/builds/11.3/cdt-11.3.0-m2a/" />
-			<unit id="org.eclipse.cdt.feature.group" version="0.0.0" />
-			<unit id="org.eclipse.cdt.platform.feature.group" version="0.0.0" />
+			<repository location="https://download.eclipse.org/tools/cdt/builds/11.3/cdt-11.3.0-m2a/"/>
+			<unit id="org.eclipse.cdt.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.cdt.platform.feature.group" version="0.0.0"/>
 		</location>
-		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/tools/orbit/downloads/drops/R20230302014618/repository/" />
-			<unit id="com.google.guava" version="0.0.0" />
-			<unit id="org.yaml.snakeyaml" version="0.0.0" />
-			<unit id="org.mockito.mockito-core" version="0.0.0"/>
+		<location includeDependencyDepth="none" includeDependencyScopes="compile" includeSource="true" label="DirectFromMaven" missingManifest="error" type="Maven">
+			<dependencies>
+				<dependency>
+					<groupId>com.google.guava</groupId>
+					<artifactId>guava</artifactId>
+					<version>32.1.1-jre</version>
```

Comment:
I would get it from here https://download.eclipse.org/tools/orbit/simrel/maven-osgi/milestone/latest/index.html